content/en/docs/Installation/trustedboot.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Trusted Boot Installations"
+linkTitle: "Trusted Boot"
+weight: 6
+date: 2022-11-13
+description: >
+---
+
+"Trusted Boot" is a combination of technologies that allows us to guarantee that a system was not tampered with, and the user-data is protected by cold attacks, it is composed by FDE, Secure Boot and Measured Boot.
+
+If you want to learn more on what Trusted Boot is and how it works, see the [Trusted Boot Architecture]({{< relref "../architecture/trustedboot" >}}) page. This page describes how to enable Trusted Boot support in Kairos.
+
+## Enable Trusted Boot in Kairos
+
+Kairos supports Trusted boot by generating special installable medium, and we boot into what we call “UKI-mode”. [UKI](https://uapi-group.org/specifications/specs/unified_kernel_image/) is a specific file format tailored to achieve a tamper-proof system and encryption of user-data bound to the silicon by relying on HW capabilities. This feature is optional and works alongside how Kairos works.
+
+> UKI stands for “Unified Kernel Images”. UKI files are a single, fat binary that encompasses the OS and the needed bits in order to boot the full system with a single, verified file. You can read technical details here: Brave New Trusted Boot World.
+
+## Requirements
+
+- Secure boot available in the system
+- The Hardware should have a TPM chip or fTPM enabled
+- The Hardware should be capable of booting large EFI files (>32MB)
+
+## Usage
+
+In order to boot into UKI mode, you need to build a special ISO file with the UKI files. To build this medium you have to generate a set of keypairs first: one for the Secure boot and one for the PCR policies required to encrypt the user-data.
+
+Any change, or upgrade of the node to a new version of the OS requires those assets to be regenerated with these keypairs, including the installer ISO, and the EFI files used for upgrading. The keys are used to *sign* and *verify* the EFI files, and the PCR policies are used to *encrypt* and *decrypt* the user-data, and thus are required to be the same for the whole lifecycle of the node.
+
+The steps below will guide you into generating the installable assets, and how to re-generate the assets to upgrade the node to a new version of the OS.
+
+### Requirements
+
+- Docker
+- Git
+- A Linux machine with KVM (for testing the images locally)
+
+### Build the container image used to generate keys and installable medium
+
+```bash
+# Build the container image that will be used to generate the keys and installable medium
+git clone https://github.com/kairos-io/enki.git
+cd enki
+docker build -t enki --target tools-image .
+```
+
+### Key generation
+
+To generate the Secure boot certificates and keys run the following commands:
+
+```bash
+# Generate the keys
+docker run -v $PWD/keys:/work/keys -ti --rm enki genkey "Kairos" -o /work/keys
```

Comment:
Would mention that they should use their own name there for the certs as those show up when listing the efivars for the database and in the signed files and such